packages/editor/src/components/post-type-support-check/test/index.js

```diff
@@ -10,27 +10,26 @@ import { PostTypeSupportCheck } from '../';
 
 describe( 'PostTypeSupportCheck', () => {
 	it( 'renders its children when post type is not known', () => {
-		let postType;
-		const tree = create(
-			<PostTypeSupportCheck postType={ postType } supportKeys="title">
+		const { container } = render(
+			<PostTypeSupportCheck postType={ undefined } supportKeys="title">
 				Supported
 			</PostTypeSupportCheck>
 		);
 
-		expect( tree.toJSON() ).toBe( 'Supported' );
+		expect( container ).toHaveTextContent( 'Supported' );
```

Comment:
As per the [official recommendations](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-container-to-query-for-elements) to use `screen` for querying elements when possible, could we perhaps go with something like:

```suggestion
		expect( screen.getByText( 'Supported' ) ).toBeVisible();
```

Reply:
There's no actual element, just an RTL wrapper, and the text. This is the reason I went with `toHaveTextContent`. Similar to #44818.

Reply:
Makes sense 👍 Feel free to ignore my suggestions in that case.